src/views/ApartmentApp/components/StaffMenu/components/ApplicationTable/components/ApplicationRow/index.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { Collapse, IconButton, TableCell, TableRow } from '@material-ui/core/';
+import KeyboardArrowDownIcon from '@material-ui/icons/KeyboardArrowDown';
+import KeyboardArrowUpIcon from '@material-ui/icons/KeyboardArrowUp';
+import ApplicantSubTable from './components/ApplicantSubTable';
+import HallSubTable from './components/HallSubTable';
+
+const ApplicationRow = ({ applicationDetails, labelId }) => {
+  const [open, setOpen] = React.useState(false);
+
+  return (
+    <React.Fragment>
+      <TableRow key={applicationDetails.ApplicationID} onClick={() => setOpen(!open)}>
+        <TableCell align="center" component="th" id={labelId} scope="row">
```

Comment:
Is there a reason the `ApplicationID` cell should be a Table Header component? It seems to me like this should be a regular table data cell.

Reply:
tbh, I did that because that is what they did in the examples in the Material-UI.com documentation, and I didn't fully understand what it meant